### PR TITLE
feat: support arm64 simulator WPB-1157

### PR DESCRIPTION
**Pull request by johnxnguyen**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1157" title="WPB-1157" target="_blank">TaskWPB-1157</a>  Update Down framework for Apple silicon support for simulator
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Add support for arm64 simulator:

- bump Down to version that supports arm64 simulator
- add arm64 as valid simulator architecture 

### Testing

Tested manually by building the app for simulator on Apple silicon.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by netbe** [comment]
@johnxnguyen hard to see in the pbxproj, but did you remove the excluded archs in all the targets too (tests and host tests) ?
+ put back build debug architecture only yes 
release no

**Review by David-Henner — APPROVED** [review]

**Comment by johnxnguyen** [comment]
@netbe for every target I set:
- build only active arch true for debug, false for release
- removed all arm64 for excluded archs

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -10452,10 +10496,15 @@
...
 			buildSettings = {
+				"ARCHS[sdk=iphonesimulator*]" = (
+					x86_64,
+					arm64,
+				);
 				CODE_SIGN_IDENTITY = "";
 				"CODE_SIGN_IDENTITY[sdk=macosx*]" = "Apple Development";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator14.4]" = i386;
```
Comment (netbe) [inline wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: does this still make sense ?

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -10437,10 +10476,15 @@
...
```
Comment (netbe) [inline wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: same

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (johnxnguyen) [reply wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: I don't know, what do you think?

**Comment by netbe** [comment]
@johnxnguyen I don't think we have to explicitly set that. We can rely on standard settings or if needed used the xcconfig files we have already which should set the proper values everywhere

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (netbe) [reply wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: I would try removing it and see if it works. Neiher the 14.4 nor the arch makes sense to me

**Comment by johnxnguyen** [comment]
> @johnxnguyen I don't think we have to explicitly set that. We can rely on standard settings or if needed used the xcconfig files we have already which should set the proper values everywhere

@netbe  I agree and started to do this, but quickly discovered that the configs of different projects is a bit of a mess. I suggest we create a separate task to simplify/unify the project configs.

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj (hunk 1 above)
Comment (johnxnguyen) [reply wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: removed

File: wire-ios/Wire-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (johnxnguyen) [reply wire-ios/Wire-iOS.xcodeproj/project.pbxproj]: removed

**Review by netbe — APPROVED** [review]
ok, @johnxnguyen  can you create a ticket for the config ? 
This is compiled for 14.2 right?

**Comment by johnxnguyen** [comment]
> ok, @johnxnguyen can you create a ticket for the config ?
> This is compiled for 14.2 right?

@netbe I created the ticket, shared it in Wire. Yes I built using Xcode 14.2.

**Pull request merged**